### DISCUSSION TITLE
Consultas Enunciado TPA3

### DISCUSSION THREAD

**Discussion by casatom** [dds-jv/foro#288] [TPA - 2021] [answered]
Buenas, tenemos algunas dudas sobre esta entrega:

- Las personas que tienen intención de adoptar, que preguntas se le debe hacer? preguntar por las características? o hacer las mismas preguntas que define la organización?

- Como deberíamos matchear las mascotas en adopción y las personas que quieren adoptar, para dar las recomendaciones? Sabemos que por las preferencias y comodidades, pero que formato o preguntas deberíamos hacer, las de las preguntas de la organización o las características de las mascotas?

**Comment by gastonprieto** [comment, marked as answer]
buenas @casatom 

- serían las mismas preguntas que respondio el que intento dar en adopción, pero vista desde otro enfoque, esto te va a dar la posibilidad de hacer el matcheo

- sería en principio en base a las preguntas generales + las de la organización.
